templates/README.md

```diff
@@ -10,9 +10,9 @@ To use this template, view as plain text (it contains comments), then edit as ap
 <!-- fill in the blanks -->
 # Create an Offline First Shopping List with [technologies]
 
-In this code pattern, we will create an Offline First shopping list. Shopping List is an Offline First demo Progressive Web App built using [technologies]. [This app is part of a series of Offline First demo apps, each built using a different stack.](https://github.com/ibm-watson-data-lab/shopping-list).
+This code pattern is a reference implementation of an Offline First shopping list app, built as a Progressive Web App using [technology](technologyURL). [This app is part of a series of Offline First demo apps, each built using a different stack.](https://github.com/ibm-watson-data-lab/shopping-list).
```

Comment:
Maybe add a comment before this paragraph:

    <!--Adjust this paragraph if your implementation is NOT a PWA. -->